addons/mail_bot_hr/views/res_users_views.xml

```diff
@@ -11,14 +11,27 @@
             </field>
         </record>
 
-        <record id="res_users_view_form_profile" model="ir.ui.view">
+        <record id="res_users_view_form_preferences" model="ir.ui.view">
             <field name="name">res.users.preferences.form.inherit</field>
             <field name="model">res.users</field>
-            <field name="inherit_id" ref="hr.res_users_view_form_profile" />
+            <field name="inherit_id"
+                   ref="mail_bot.res_users_view_form_preferences" />
             <field name="arch" type="xml">
+                <field name="odoobot_state" position="before">
+                    <field name="can_edit" invisible="1"/>
+                </field>
                 <xpath expr="//field[@name='odoobot_state']" position="attributes">
                     <attribute name="attrs">{'readonly': [('can_edit', '=', False)]}</attribute>
                 </xpath>
+            </field>
+        </record>
+
+        <record id="res_users_view_form_profile" model="ir.ui.view">
```

Comment:
Why two views ? Seems res_users_view_form_profile inherit from res_users_view_form_simple_modif which inherit from view_users_form_simple_modif; same for res_users_view_form_preferences which inherit from view_users_form_simple_modif_mail which inherit from view_users_form_simple_modif .

Seems everything could be simplified, unless there are some strange implications ?

Reply:
The key is that odoobot_state is not declared in hr, so a hr view can't be used in the inherit. And the sheet/notification_alert part is left as it was, untouched, inheriting from hr.res_users_view_form_profile.